cluster/juju/layers/kubernetes-worker/templates/nfs-provisioner.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: default
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: fuseim.pri/ifs
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: nfs-client-provisioner
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: nfs-client-provisioner
+    spec:
+      containers:
+        - name: nfs-client-provisioner
+          image: quay.io/external_storage/nfs-client-provisioner:latest
```

Comment:
This is an amd64 image, meaning the NFS relation won't work on arm64, s390x, or others.

Let me think about this some more. I'll come back to this PR tomorrow.

Reply:
Good point. I see an arm version, but nothing for s390x or others.. 

quay.io/external_storage/nfs-client-provisioner-arm:latest

Reply:
Discussed this offline, we settled on just shipping amd64 support for now.